Neos.Flow/Classes/Command/PackageCommandController.php

```diff
@@ -99,27 +99,6 @@ public function createCommand(string $packageKey, string $packageType = PackageI
         $this->outputLine('Created new package "' . $packageKey . '" at "' . $package->getPackagePath() . '".');
     }
 
-    /**
-     * Delete an existing package
-     *
-     * This command deletes an existing package identified by the package key.
-     *
-     * @Flow\FlushesCaches
-     * @param string $packageKey The package key of the package to create
-     * @return void
-     */
-    public function deleteCommand(string $packageKey)
-    {
-        if (!$this->packageManager->isPackageAvailable($packageKey)) {
-            $this->outputLine('The package "%s" does not exist.', [$packageKey]);
-            $this->quit(1);
-        }
-        $this->packageManager->deletePackage($packageKey);
-        $this->outputLine('Deleted package "%s".', [$packageKey]);
-        Scripts::executeCommand('neos.flow:cache:flush', $this->settings, false);
-        $this->sendAndExit(0);
-    }
-
```

Comment:
The `PackageCommandController` now complains about missing methods in `createCommand()` and `listCommand()`

Reply:
oh? ok, will have a look.